test/10_migrations.ts

```diff
@@ -1,10 +1,24 @@
 import { strict as assert } from 'assert';
 import fs from 'fs';
 import _ from 'lodash';
+import { execSync } from 'node:child_process';
 import path from 'path';
-import parser from 'libpg-query';
 import configJson from '../config';
 
+// Validate SQL files using squawk
+function validateSql(files: string[]): void {
+	try {
+		execSync(
+			`npx squawk --pg-version=15.0 --dump-ast parsed ${files.join(' ')}`,
+			{
+				stdio: ['ignore', 'ignore', 'pipe'],
+			},
+		);
+	} catch (e) {
```

Comment:
If it fails, will it highlight the name of the migration?

Reply:
Responed in https://github.com/balena-io/balena-api/pull/4553#discussion_r1293174594